pkg/nvlib/info/info.go

```diff
@@ -27,15 +27,22 @@ import (
 )
 
 type infolib struct {
+	logger basicLogger
+	Properties
+	Resolver
+}
+
+type info struct {
 	root      root
 	nvmllib   nvml.Interface
 	devicelib device.Interface
 }
 
+var _ Properties = &info{}
```

Comment:
I'm starting to get lost in all the differences in naming and levels of indirection.